frontend/src/features/projects/common/hooks/useProject.tsx

```diff
@@ -34,10 +34,11 @@ const useProject = () => {
   return {
     goToStepByCode: (statusCode: string) => {
       const status: IStatus | undefined = _.find(workflowStatuses, { code: statusCode });
-      navigate(`..${status?.route}?projectNumber=${project?.projectNumber}`);
+      navigate(`/dispose${status?.route}?projectNumber=${project?.projectNumber}`);
+    },
+    goToDisposePath: (path: string) => {
+      navigate(`/dispose${path}?projectNumber=${project?.projectNumber}`);
     },
```

Comment:
Thanks Braden!

Reply:
I found another situation when editing projects that are in either a "Submitted" or "Submitted Exemption" status, and trying to click the "Add more properties" button, it's currently going to the wrong url: http://localhost:3000/disposeproperties/update?projectNumber=SPP-18073